src/components/Cards/HorizontalCard/HorizontalCard.jsx

```diff
@@ -1,5 +1,64 @@
-const HorizontalCard = () => {
-  return <div>HorizontalCard</div>;
+import { useState } from "react";
+import { Link } from "react-router-dom";
+import useClickOutside from "../../../hooks/useClickOutside";
+
+import { useDataContext, useGlobalContext } from "../../../context";
+import { removeFromHistory } from "../../../services/history";
+
+// Image
+import { videoImage } from "../../../utils/imageGenerator";
+
+import "./horizontal-card.scss";
+
+// Icons
+import { BsPlayCircle } from "react-icons/bs";
+import { BiDotsVertical } from "react-icons/bi";
+import { RiDeleteBin7Fill } from "react-icons/ri";
+import { CgPlayListAdd } from "react-icons/cg";
+
+const HorizontalCard = ({ _id, title, categoryName, creator, type }) => {
+  const [open, setOpen] = useState(false);
+  const domNode = useClickOutside(() => setOpen(false));
+
+  const { dataDispatch } = useDataContext();
+  const {
+    globalHandlers: { openAlert },
+  } = useGlobalContext();
+
+  const handleDelete = () => {
+    if (type === "history") {
+      removeFromHistory(_id, dataDispatch, openAlert);
```

Comment:
You can extract "history" as constant like other constants you did

Reply:
okay, will make it a constant in playlist PR.